src/utils/QCCalculator.cpp

```diff
@@ -396,29 +391,48 @@ class TOPPQCCalculator :
     at.colTypes.push_back("MS:1000285");
     UInt max = 0;
     Size below_10k = 0;
-    for (Size i = 0; i < exp.size(); ++i)
-    {
-      if (exp[i].getMSLevel() == 1)
+    std::vector<OpenMS::Chromatogram> chroms = exp.getChromatograms();
+    if (chroms.size() == 1) //real TIC from the mzML
```

Comment:
I guess if there is more than one spectrum and exactly one chromatogram, we can assume that the one chromatogram is the TIC. It would be nice though to check with a CV param or something